### PR TITLE
added november ndvi for ethiopia/oromia, changed default niger obs

### PR DESCRIPTION
**Pull request by kgraaf**

### CODE REVIEW

File: fbfmaproom/fbfmaproom-sample.yaml [hunk 1]
```diff
@@ -1228,7 +1252,7 @@ countries:
             defaults:
               predictors:
                 - pnep
-                - chirp-spi-jj
+                - enacts-mon-spi-jas
```
Comment (aaron-kaplan) [inline fbfmaproom/fbfmaproom-sample.yaml:1255]: Dan said JJ, not JAS.

**Comment by kgraaf** [comment]
Can fix

**Comment by aaron-kaplan** [comment]
Your seasons are centered on April 9 instead of April 16. Consequently, the table rows don't line up.

image

Are you still working on this or have you taken the afternoon off?

**Comment by aaron-kaplan** [comment]
I'm fixing it.

**Pull request merged**